docs/apache-airflow/howto/operator/python.rst

```diff
@@ -129,19 +129,18 @@ If you want the context related to datetime objects like ``data_interval_start``
 .. _howto/operator:ShortCircuitOperator:
 
 ShortCircuitOperator
-========================
+====================
```

Comment:
Should we change this section title to `@task.short_circuit`?

Reply:
I thought about it and then didn't to be consistent with the rest of the doc.  Oh but I did forget to add the "warning" block mentioning to use the decorator instead of the classic operator like the others. I'll fix that for sure.

Reply:
I do think this how-to doc should be refactored a little to really emphasize the decorator use or even specifically having a `howto/decorators/python` doc instead.

Reply:
There is #25319 for this. Maybe add a todo item there?

Reply:
Solid plan to me.